dockerfiles/Dockerfile.package

```diff
@@ -1,24 +1,15 @@
-ARG RESTY_IMAGE_BASE="ubuntu"
-ARG RESTY_IMAGE_TAG="bionic"
+ARG RESTY_IMAGE_BASE="alpine"
+ARG RESTY_IMAGE_TAG="3.12"
 ARG DOCKER_KONG_SUFFIX

```

Comment:
Dockerfile.package was rewritten to avoid using the FPM image, which doesn't exist for s390x.  However fpm is not needed in order to create the alpine apk needed for Linux for Health.  For general use, we will need to create an FPM image on s390x to support other install types (e.g rpm).

Reply:
Built FPM for s390x - resolved.